src/sentinel.c

```diff
@@ -3746,7 +3746,7 @@ void sentinelCommand(client *c) {
             "    failover.",
             "CONFIG SET param value [param value ...]",
             "    Set a global Sentinel configuration parameter.",
-            "CONFIG GET <param> [param param param ...]",
+            "CONFIG GET <param> [param param ... param]",
```

Comment:
It can be any number of params, so open-ended seems better to me.

(The style with an end value is good in sequences like `1, 2, ..., 10`, when there is a specific last element.)

Btw we don't use the angular brackets in the other syntax examples like CONFIG SET tw lines above.

```suggestion
            "CONFIG GET param [param ...]",
```